### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by kaijli**
updated sample data and general documentation

### CODE REVIEW

File: docs/metat_workflow2024.png
Comment (aclum) [inline docs/metat_workflow2024.png:1]: Can you make this look more like https://nmdc-workflow-documentation.readthedocs.io/en/latest/chapters/3_MetaGAssemly_index.html where its clearer that bbmap is done on the contigs from assembling the transcripts? That mapping file plus the GFF then inptu to JGI readcount.

Comment (kaijli) [reply docs/metat_workflow2024.png:1]: mt_workflow
how about this one?

Comment (aclum) [reply docs/metat_workflow2024.png:1]: yes, this looks good.

**Review by aclum — COMMENTED** [review]
In some repos you hyperlink the docker images and some you don't. I do like the hyperlinked version better.

**Comment by kaijli** [comment]
> In some repos you hyperlink the docker images and some you don't. I do like the hyperlinked version better.

so this repo is good, right?

**Comment by aclum** [comment]
There is no hyperlinking in the README.md for the docker images, just in the index.rst. If you could add the hyperlinks to the README.md we are good to go here.

**Review by aclum — APPROVED** [review]

**Pull request merged**